tests/dummy/app/routes/components-examples/flexberry-groupedit/model-update-example.js

```diff
@@ -0,0 +1,33 @@
+import EditFormRoute from 'ember-flexberry/routes/edit-form';
+
+export default EditFormRoute.extend({
+  /**
+    Name of model projection to be used as record's properties limitation.
+
+    @property modelProjection
+    @type String
+    @default 'ManualModelUpdateView'
+   */
+  modelProjection: 'ManualModelUpdateView',
+
+  /**
+    Name of model to be used as form's record type.
+
+    @property modelName
+    @type String
+    @default 'components-examples/flexberry-groupedit/settings-example/aggregator'
+   */
+  modelName: 'components-examples/flexberry-groupedit/settings-example/aggregator',
+
+  /**
+    Returns model related to current route.
+
+    @method model
+   */
+  model(params) {
+    let store = this.get('store');
+
+    // Empty aggregator without details.
+    return store.createRecord('components-examples/flexberry-groupedit/settings-example/aggregator', {});
```

Comment:
To my mind this example is an extending of setting-example (it will be too difficult for undestanding to place it on setting-example page). So I reused model but created separate projection so this examples could exist together.